connect.go

```diff
@@ -290,7 +291,7 @@ func (c *Conn) destroy(o directObj) error {
 	if err != nil {
 		return err
 	}
-	if len(msg.Data) > 0 {
+	if len(msg.Data) > 0 && !strings.Contains(string(msg.Data), "not exist") {
```

Comment:
Ideally we'd want all errors organised in one place, and each error to have a unique "return code" and not rely on strings. But that's a story for a different day 😉

Reply:
for that point it is ok because currently the broker don't send status codes